src/NLog/LayoutRenderers/Contexts/TraceActivityIdLayoutRenderer.cs

```diff
@@ -46,7 +46,6 @@ namespace NLog.LayoutRenderers
     /// A renderer that puts into log a System.Diagnostics trace correlation id.
     /// </summary>
     [LayoutRenderer("activity-id")]
-    [LayoutRenderer("activityid")]
```

Comment:
I prefer to keep `activityid` as the default-name.

Reply:
But I also recognize that it overlapse with https://github.com/NLog/NLog.DiagnosticSource

Reply:
it's called `${activity}` there?

Reply:
Yes you are right, then no issue. Guess I was forseen when creating NLog.DiagnosticSource :)

Reply:
(It would be hard if we ${activityid} and ${activity-id}. The difference is hard to communicate when speaking to others :P)

Reply:
Fixed this. I consistently used the ones without the dash, but move now to the version from https://nlog-project.org/config/?tab=layout-renderers